Assets/Scripts/CameraShake.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using EasyButtons;
+
+/****************************************
+ *  Camera will simulate a shaking motion. 
+ *  Need to connect the player sprite to
+ *  to the CameraShake.cs script.
+ ****************************************/
+public class CameraShake : MonoBehaviour{
+    public static CameraShake instance;
+    public Transform player;
+
+    [SerializeField]
+    private float maxOffset, // Controls the range of GetPerlinNoise()
```

Comment:
I think these should be customizable. one option is StartShake() having optional arguments.
another option is to make a scriptable object that lets you create different camera shake "profiles"